docker/layotto-file/config_file.json

```diff
@@ -0,0 +1,53 @@
+{
+    "servers": [
```

Comment:
How about naming the path as `docker/layotto-minio/config_minio.json`, which is consistent with the style of `docker/layotto-redis/config_redis.json`